src/ifk_scb_compilations/objects/passenger_transport.py

```diff
@@ -0,0 +1,127 @@
+"""Inputs for request, and analysis."""
+
+from dataclasses import dataclass
+
+import matplotlib.pyplot as plt
+import pandas as pd
+
+
+@dataclass
+class Request_input:
+    """Dataclass for passenger transport query.
+
+    Query info can be found at url.
+    """
+
+    url = "https://api.scb.se/OV0104/v1/doris/sv/ssd/START/MI/MI0107/TotaltUtslappN"
+    query = {
+        "query": [
+            {
+                "code": "Vaxthusgaser",
+                "selection": {"filter": "item", "values": ["CO2-ekv."]},
+            },
+            {
+                "code": "Sektor",
+                "selection": {"filter": "item", "values": ["0.2", "0.4", "8.0", "5.0"]},
+            },
+        ],
+        "response": {"format": "json"},
+    }
+
+
+class Analysis:
+    """Container for plotting data corresponding to fetch spec by Request_input."""
+
+    def __init__(self, request_output: dict) -> None:
+        """Initialization.
+
+        Args:
+            request_output (dict): Output from scb api response.
```

Comment:
```suggestion
            request_output: Output from scb api response.
```

I think this is unnecessary since the type is annotated in the function parameter. If we generate docs the types will be included.

Reply:
should be updated.